src/test/java/com/microsoft/azure/proton/transport/proxy/impl/ProxyHandlerImplTest.java

```diff
@@ -56,11 +56,11 @@ public void testValidateProxyResponseOnSuccess(int httpCode) {
 
     @Test
     public void testValidateProxyResponseOnFailure() {
-        final String failResponse = "HTTP/1.1 407 Proxy Auth Required\r\n" +
-                "Connection: close\r\n" +
-                "Proxy-Authenticate: Basic realm=\"FiddlerProxy (user: 1, pass: 1)\"\r\n" +
-                "Content-Type: text/html\r\n" +
-                "<html><body>[Fiddler] Proxy Authentication Required.<BR></body></html>\r\n\r\n";
+        final String failResponse = String.join("\r\n", "HTTP/1.1 407 Proxy Auth Required",
+            "Connection: close",
+            "Proxy-Authenticate: Basic realm=\\\"FiddlerProxy (user: 1, pass: 1)\\",
+            "Content-Type: text/html",
+            "<html><body>[Fiddler] Proxy Authentication Required.<BR></body></html>\r\n", "\r\n");

```

Comment:
Would this add one more `\r\n` than the original because we have a trailing `\r\n` in `</html>`?

Reply:
Noted, removed all the unnecessary trailing new line delimiters for all the unit tests from, `ProxyHandlerImplTest.java` , expect for `testCreateProxyRequest` as trailing new line  is appended in `ProxyHandlerImpl.createProxyRequest` method.

  ```
@Override
    public String createProxyRequest(String hostName, Map<String, String> additionalHeaders) {
        .......
        connectRequestBuilder.append(NEW_LINE);
        return connectRequestBuilder.toString();
    } 
```